nextbike/datapreparation/data_clean.py

```diff
@@ -64,11 +61,11 @@ def data_cleaning(p_df_original):
     # Only keep trips, which where merged correctly
     df_merged = df_merged[df_merged["b_number_start"] == df_merged["b_number_end"]]
 
-    df_merged.drop(
+    df_merged = df_merged.drop(
         ["key_0",
-         "b_number_end"], axis=1, inplace=True
+         "b_number_end"], axis=1
```

Comment:
reindexing earlier instead of drop key_0 would be better

Reply:
I just removed the inplace=True option here :D
I thought key_0 is something produced by the merge? How should we drop this before merging then? 🤔